source/includes/lifecycle-schedule-callout.rst

```diff
@@ -0,0 +1,6 @@
+.. important::
+
+   MongoDB maintains a support policy to ensure compatibility between MongoDB
+   Server and drivers for three years after the server version's end of
```

Comment:
```suggestion
   MongoDB maintains a support policy to ensure compatibility between the MongoDB
   Server and drivers for three years after the server version's end of
```

Reply:
I can change it to "the Server and the drivers" (requires parallelism) across PRs.